services/web/server/src/simcore_service_webserver/wallets/_events.py

```diff
@@ -23,11 +29,30 @@ async def _auto_add_default_wallet(
             product_name=product_name,
         )
 
+        if extra_credits:
+            user = await get_user_name_and_email(app, user_id=user_id)
+            await add_credits_to_wallet(
+                app,
+                product_name=product_name,
+                wallet_id=wallet.wallet_id,
+                wallet_name=wallet.name,
+                user_id=user_id,
+                user_email=user.email,
+                osparc_credits=extra_credits,  # type: ignore
+                payment_id="INVITATION",  # TODO: invitation id???
```

Comment:
Could you please assign this to a constant that we know of and can be easily replaced in the future if used elsewhere.
@matusdrobuliak66 have we have something like that for the payment_id?

Reply:
We do not, currently payment_transaction_id is in the resource usage tracker is string or NULL. It is just an informative field with additional information about payment (it is just informative and currently not used for anything, maybe in the future we can introduce payment type, but for now it is okay when you send this string. (we can make migration later when we put more thinking what we want to use it fore)

Reply:
For the moment invitations are ephemeral. I cannot identify an invitation once it is consumed.
This might change though ... because probably we would want to trace this back to the guest (i.e. who authorized this invitation)